tests/test_diffraction_objects.py

```diff
@@ -702,3 +702,66 @@ def test_copy_object(do_minimal):
     do_copy = do.copy()
     assert do == do_copy
     assert id(do) != id(do_copy)
+
+
+@pytest.mark.parametrize(
+    "starting_all_arrays, scalar_to_add, expected_all_arrays",
+    [
+        # Test scalar addition to xarray values (q, tth, d) and expect no change to yarray values
+        (  # C1: Add integer of 5, expect xarray to increase by by 5
+            np.array([[1.0, 0.51763809, 30.0, 12.13818192], [2.0, 1.0, 60.0, 6.28318531]]),
+            5,
+            np.array([[1.0, 5.51763809, 35.0, 17.13818192], [2.0, 6.0, 65.0, 11.28318531]]),
+        ),
+        (  # C2: Add float of 5.1, expect xarray to be added by 5.1
+            np.array([[1.0, 0.51763809, 30.0, 12.13818192], [2.0, 1.0, 60.0, 6.28318531]]),
+            5.1,
+            np.array([[1.0, 5.61763809, 35.1, 17.23818192], [2.0, 6.1, 65.1, 11.38318531]]),
+        ),
+    ],
+)
+def test_addition_operator_by_scalar(starting_all_arrays, scalar_to_add, expected_all_arrays, do_minimal_tth):
+    do = do_minimal_tth
+    assert np.allclose(do.all_arrays, starting_all_arrays)
+    do_sum_RHS = do + scalar_to_add
+    do_sum_LHS = scalar_to_add + do
+    assert np.allclose(do_sum_RHS.all_arrays, expected_all_arrays)
+    assert np.allclose(do_sum_LHS.all_arrays, expected_all_arrays)
+
+
+@pytest.mark.parametrize(
+    "LHS_all_arrays, RHS_all_arrays, expected_all_arrays_sum",
+    [
+        # Test addition of two DO objects, expect combined xarray values (q, tth, d) and no change to yarray
```

Comment:
as written, this is wrong.  The add functionality operates on the yarray, so xarrys should be unaffected and yarray values (`all_arrays[:,0]`).   So this test appears to be wrong.

Reply:
done